### ISSUE TITLE
Window title does not update when clicking on Volumes tab

### ISSUE THREAD

**Issue by nmbell** [docker/for-win#12616]
- [X] I have tried with the latest version of Docker Desktop
  - [X] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID: n/a

### Actual behavior
When clicking on the "Volumes" tab, the window title does not update.

### Expected behavior
When clicking on the "Volumes" tab, the window title changes to e.g. "Volumes list".

### Information
  - Windows Version:
```
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.19044 N/A Build 19044
```
  - Docker Desktop Version:
```
Client:
 Cloud integration: v1.0.22
 Version:           20.10.13
 API version:       1.41
 Go version:        go1.16.15
 Git commit:        a224086
 Built:             Thu Mar 10 14:13:04 2022
 OS/Arch:           windows/amd64
 Context:           default
 Experimental:      true

Server: Docker Desktop 4.6.0 (75818)
 Engine:
  Version:          20.10.13
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.15
  Git commit:       906f57f
  Built:            Thu Mar 10 14:06:05 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.5.10
  GitCommit:        2a1d4dbdb2a1030dc5b01e96fb110a9d9f150ecc
 runc:
  Version:          1.0.3
  GitCommit:        v1.0.3-0-gf46b6ba
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```
  - WSL2 or Hyper-V backend?  WSL2
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM: No

### Output of `& "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check`
n/a

### Steps to reproduce the behavior
1. Reset to factory settings.
2. Click on any tab other than "Volumes".
3. Note the window title.
4. Click on the "Volumes" tab.
5. Note the window title has not changed.

**Comment by nmbell** [comment]
Attaching a screen recorder gif:
![docker window title](https://user-images.githubusercontent.com/23632110/159415651-8d3c051e-e8f3-418c-aef0-dd97c512cec0.gif)

**Comment by mat007** [comment]
Good catch, we’ll fix this for the next release.
Thanks!

**Comment by ebriney** [comment]
Fixed in 4.7.0 release
